opencti-platform/opencti-front/src/private/components/common/containers/ContainerHeader.jsx

```diff
@@ -1091,6 +1099,16 @@ const ContainerHeader = (props) => {
                 {React.cloneElement(PopoverComponent, { id: container.id })}
               </Security>
             )}
+            {isFABReplaced && (
+              <Security needs={[KNOWLEDGE_KNENRICHMENT]}>
+                <StixCoreObjectEnrichment
+                  stixCoreObjectId={container.id}
+                  displayEnrichment={displayEnrichment}
+                  handleOpenEnrichment={handleOpenEnrichment}
+                  handleClose={handleCloseEnrichment}
+                />
+              </Security>
+            )}
             {EditComponent}
           </div>
         </div>
```

Comment:
`ContainerHeader.jsx` is used by other containers such as Cases. 
This could be the reason why we have `Update` button in Cases